README.md

```diff
@@ -1,193 +1,198 @@
-Visualization: A Grafana instance configured to talk to the Prometheus, Loki, and Tempo instances makes it possible to query and visualize the metrics, logs, and traces data. 
+- Logs: Each tier of the TNS app writes logs to standard output or standard error. These logs are captured by Kubernetes and then collected by the Grafana Agent, which forwards them on to Loki for storage.
 
+- Traces: Each tier of the TNS app sends traces in Jaeger format to the Grafana Agent, which then converts them to OTel format and forwards them to Tempo for storage.
 
-## Prerequisites
+- Visualization: A Grafana instance configured to talk to the Prometheus, Loki, and Tempo instances makes it possible to query and visualize the metrics, logs, and traces data.
 
-Before you begin, install and configure the following software applications.
+## Prerequisites
 
 ### Docker
 
-Verify you have Docker installed. For download and installation instructions, click [here](https://docs.docker.com/install/).
+Verify you have Docker installed. Click [here](https://docs.docker.com/install/) for installation instructions.
 
-### K3D
+**Note:** Ensure that your Docker daemon has a minimum of 2.5 GB of total memory available for all pods in this demo to be scheduled.
 
-To run the TNS demo, you need a Kubernetes cluster. The cluster creation script uses `k3d` which runs as a single node cluster inside Docker. The specific version to use depends on your operating system:
-- For Ubuntu 20.04 and lower, use (https://github.com/rancher/k3d/releases/tag/v3.2.0).
-- For Ubuntu 21.10 and higher (which has cgroups v2 enabled by default, and k8s fails to start thinking cgroups are not available), use (https://github.com/rancher/k3d/releases/tag/v5.0.0)
+### kubectl
 
-**Note:** Ensure that your Docker daemon has a minimum of 2.5 GB of total memory available for all pods in this deployment to be scheduled.
+This demo uses `kubectl` to interact with the Kubernetes clusters. Click [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for installation instructions.
 
-You can also run the TNS demo without Kubernetes. Click [here](https://github.com/grafana/tns/blob/main/production/docker-compose/README.md).
- for more information.
+### Kubernetes cluster
 
-### kubectl
+These instructions outline how to run the application inside a single-node Kubernetes cluster running inside of Docker via [`k3d`](https://k3d.io/).
 
-The TNS demo uses `kubectl` to interact with the Kubernetes clusters. Click [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for `kubectl` installation instructions.
+If you would like to you a Kubernetes cluster other than a local `k3d` one, you will need to modify the instructions to match your environment.
 
-### Tanka
+If you would like to run the TNS demo using Docker Compose rather than Kubernetes, see instructions [here](/production/docker-compose/README.md).
```

Comment:
Too many sentences starting with "If you would like to". We should keep the earlier bulleted list here.

Reply:
Removed the "If you would like" in https://github.com/grafana/tns/pull/36/commits/ce7c172a90d7431c477aa3b65e4b5c1eb8e722b8